app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -0,0 +1,185 @@
+<resources>
+    <string name="app_name" translatable="false">PeakAR</string>
+
+    <!-- Google -->
+    <!-- Sign-in status messages -->
+    <string name="mountain_info" translatable="false">Mont Blanc - Monte Bianco lat: 45.8325 lon: 6.8641666666667 alt: 4810 <br /><br />
```

Comment:
Cirrus complains that this is translated in German. Since it is set to false it shouldn't